src/components/common/SearchBar/Component.js

```diff
@@ -7,6 +7,7 @@ import { useHistory, useLocation } from 'react-router-dom';
 import '../../../styles/style.less';
 
 function SearchBar(props) {
+  console.log(props.newProps);
```

Comment:
Should we remove console logs before pushing to main @mhidalgo83 ?

Reply:
Yeah.